### ISSUE TITLE
is this project still supported

### ISSUE THREAD

**Issue by Fineteklabs** [sellpixels/visual-builder-docs#11]
Just curious is this project still supported?

**Comment by Mwimwii** [comment]
> Just curious is this project still supported?

Yeah also wondering. It seems that the web generator backend is also offline
